android/src/main/kotlin/com/shopify/reactnative/flash_list/AutoLayoutViewManager.kt

```diff
@@ -21,45 +24,44 @@ class AutoLayoutViewManager: ReactViewManager() {
         return REACT_CLASS
     }
 
-    override fun createViewInstance(context: ThemedReactContext): ReactViewGroup {
+    override fun getDelegate(): ViewManagerDelegate<AutoLayoutView> = mDelegate
+
+    override fun createViewInstance(context: ThemedReactContext): AutoLayoutView {
         return AutoLayoutView(context).also { it.pixelDensity = context.resources.displayMetrics.density.toDouble() }
     }
 
-    override fun getExportedCustomDirectEventTypeConstants(): MutableMap<String, Any> {
-        return MapBuilder.builder<String, Any>().put(
-                "onBlankAreaEvent",
-                MapBuilder.of(
-                        "registrationName", "onBlankAreaEvent")
-        ).build();
-    }
+    override fun getExportedCustomDirectEventTypeConstants() = mutableMapOf(
+        "onBlankAreaEvent" to mutableMapOf("registrationName" to "onBlankAreaEvent"),
```

Comment:
To be honest, I don't remember 😅. It's been a while since I've written this, but it doesn't seem to be causing any issues when I removed `topOnBlankAreaEvent`.